SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialSender.h

```diff
@@ -160,6 +161,8 @@ class SPATIALGDK_API USpatialSender : public UObject
 	UPROPERTY()
 	UActorGroupManager* ActorGroupManager;
 
+	SpatialVirtualWorkerTranslator* VirtualWorkerTranslator;
```

Comment:
As I understand, `VirtualWorkerTranslator` behaves as if it is a weak_ptr here (it's not a UPROPERTY and references a data, that unique_ptr owns in `SpatialNetDriver`). Should we make `VirtualWorkerTranslator` a shared_ptr in `SpatialNetDriver` and a weak_ptr here? That way the relation would be explicit. But there shouldn't be any bugs with the code as is though.

Reply:
I'll look into this tomorrow morning.

Reply:
After staring at this a lot, I've made the VirtualWorkerTranslator in NetDriver public. This is the same approach taken by a large number of other components in SpatialSender and SpatialReceiver, which then just access the functioanlity via the NetDriver.

I know this is tying more things to the NetDriver, but they were already tied there, and this way when we have a method for untangling the NetDriver for one thing, we can apply it as well to the LoadBalancing code.